thunder/transforms/quantization.py

```diff
@@ -294,3 +295,214 @@ def transform_traces_pre_prologue(self, prologue_trace, computation_trace, epilo
 
         new_computation_trace.set_provenance(thunder.core.trace.TraceProvenance("quant pass"))
         return prologue_trace, new_computation_trace, epilogue_trace
+
+
+class LORATransform(Transform):
+    def __init__(
+        self,
+        r: int = 0,
+        lora_alpha: int = 1,
+        lora_dropout: float = 0.0,
+        **kwargs,
```

Comment:
I don't think we want blanket kwargs here. We just use them in the transform methods because some transforms want additional keyword-args, but for the init it is better to error if we get arguments we don't know.
Maybe we should also make the other args kw-only (by inserting a `*, ` after `self`).
Also, I would likely want to support passing in a list of weight names which we want to create LoRA paths for.

Reply:
I agree to
- remove `**kwargs`
- make the arguments kw-only
Thanks for the suggestion!

Regarding the weight names, what would be the default that will be inserted?